### ISSUE TITLE
Resource Leakage Bad Practices

### ISSUE THREAD

**Issue by AngadSudan** [Mohit5Upadhyay/WeatherApplication#39]
The API keys and such information are never pushed on github to avoid Resource Leakages and is a bad practice to do so.

![image](https://github.com/user-attachments/assets/14cbdaca-22a7-4efd-8fe9-d59ce4101496)

Fixes- Add a .env file and insert the Api key

**Comment by AngadSudan** [comment]
If you liked the proposal, I'd be happy to work on it.

**Comment by Mohit5Upadhyay** [comment]
Yes , @AngadSudan  go ahead to implement and add .env file for all sensitive information
